conventions/README.md

```diff
@@ -14,6 +32,100 @@ This directory contains *approved* conventions and represents best practices in
+
+This schema extends [ERC721 Metadata JSON Schema](https://eips.ethereum.org/EIPS/eip-721), an existing well-known data model for digital assets.
+
+```
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "description": "An abstract digital asset schema.",
+  "properties": {
+    "$evidence": {
+      "description": "A URI pointing to the evidence JSON with data needed to certify this asset.",
+      "type": "string"
+    },
+    "$schema": {
+      "description": "A path to JSON Schema definition file.",
+      "type": "string"
+    },
+    "name": {
+      "description": "A property that holds a name of an asset.",
+      "type": "string"
+    },
+    "description": {
+      "description": "A property that holds a detailed description of an asset.",
+      "type": "string"
+    },
+    "image": {
+      "description": "A public property that can be a valid URI pointing to a resource with mime type image/* representing the asset to which this digital assets represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive.",
+      "type": "string"
+    }
+  },
+  "title": "Asset",
+  "type": "object",
+  "required": ["$schema"]
+}
+```
+
+A more human-friendly description of this above specification follows. This is provided to help the reader however only the above machine-readable description is authoritative.
+
+- The data model is a JSON object.
+- The JSON object MUST include a `$schema` string property on the root of the object.
+- The JSON object MAY include an `$evidence` string property on the root of the object, and if so this will be interpreted as a URI pointing to an evidence file which certifies any imprints of this digital asset.
+- The JSON object MAY include `name`, `description` and `image` string property on the root of the object, which if present will be interpreted as specified in ERC721 Metadata JSON Schema.
+- For the optional string properties above, the JSON object MUST NOT include a property on the root of the object with a non-string type using any of those names
+
+## Examples
+
+```
+{
+  "$evidence": "https://troopersgame.com/dog/evidence.json",
+  "$schema": "https://conventions.0xcert.org/86-base-asset-schema.json",
+  "description": "A weapon for the Troopers game which can severely injure the enemy.",
+  "image": "https://troopersgame.com/dog.jpg",
+  "name": "Magic Sword"
+}
+```
+
+## Rationale
+
+- Why micro-credentials
+- What are they solving
+- Typical terms
+
+## Backwards Compatibility
+
+Currently, there is no schema defined for backwards compatibility, so it's backwards-related to everything blockchain. Blockchain-wise it has no problem. As for the current academic status, please elaborate on specific standards for academic credentials in Europe, and how this convention is compatible with existing EU standards for micro-credentials.
+
+## Implementations
+
+Use of ERC-721 non-fungible tokens (NFTs)
```

Comment:
This are not actual implementation of conventions...

Reply:
These unintentional changes were removed.